lib/tools/browser/index.ts

```diff
@@ -0,0 +1,16 @@
+import browserAct from "./browserAct";
+import browserExtract from "./browserExtract";
+import browserObserve from "./browserObserve";
+import browserAgent from "./browserAgent";
+import waitTool from "./waitTool";
+
+const browserTools = {
+  browser_act: browserAct,
+  browser_extract: browserExtract,
+  browser_observe: browserObserve,
+  browser_agent: browserAgent,
+  wait: waitTool,
+};
+
+export default browserTools;
```

Comment:
optional: Perfect use case for an Agent
- https://ai-sdk-tools.dev/agents

<img width="1440" height="900" alt="Image" src="https://github.com/user-attachments/assets/a6564acb-4331-4d17-8d80-17a021b96a48" />